dkg-gadget/src/messages/misbehaviour_report.rs

```diff
@@ -29,7 +23,7 @@ where
 {
 	// Get authority accounts
 	let header = dkg_worker.latest_header.as_ref().ok_or(DKGError::NoHeader)?;
-	let current_block_number = header.number().clone();
+	let _current_block_number = header.number().clone();
```

Comment:
I think this line can be removed if we are not using it and its only fetching some data rather then setting some data. Thoughts?

Reply:
Alright